src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -152,39 +175,96 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                     <Typography variant="h2" gutterBottom>
                       {t('registration.files_and_license.files')}
                     </Typography>
+
                     {files.length > 0 && (
                       <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mb: '2rem' }}>
-                        {files.map((file) => {
-                          const associatedFileIndex = associatedArtifacts.findIndex((artifact) => {
-                            if (associatedArtifactIsFile(artifact)) {
-                              const associatedFile = artifact as AssociatedFile;
-                              return associatedFile.identifier === file.identifier;
-                            }
-                            return false;
-                          });
+                        <TableContainer component={Paper}>
+                          <Table sx={alternatingTableRowColor}>
+                            <TableHead>
+                              <TableRow>
+                                <TableCell>
+                                  <Typography fontWeight="bold">{t('common.name')}</Typography>
+                                </TableCell>
+                                <TableCell></TableCell>
+                                <TableCell>
+                                  <Typography fontWeight="bold">{t('registration.files_and_license.size')}</Typography>
+                                </TableCell>
+                                <TableCell>
+                                  <Typography fontWeight="bold">
+                                    {t('registration.files_and_license.administrative_agreement')}
+                                  </Typography>
+                                </TableCell>
+                                <TableCell>
+                                  <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                                    <Typography fontWeight="bold">{t('common.version')}</Typography>
+                                    <Typography sx={{ color: 'error.main' }}>*</Typography>
```

Comment:
Tror dette skal være lov..?
```suggestion
                                    <Typography color="error">*</Typography>
```